state-machine.md

```diff
@@ -44,8 +44,11 @@ If the execution completes successfully, changes to the state tree are saved. Ot
 ```go
 func ApplyMessage(st StateTree, msg Message) MessageReceipt {
 	st.Snapshot()
-	fromActor := st.GetActor(msg.From)
-
+	fromActor, found := st.GetActor(msg.From)
+  if !found {
```

Comment:
This indentation seems off here and elsewhere.

Reply:
There is a lot of message validation that needs to happen on the message. The existence of the from actor is just one of those. This is probably fine though.

Reply:
mind taking a stab at adding more of that validation? Its worth having here. Just submit another PR whenever